docs/api.asciidoc

```diff
@@ -174,12 +185,14 @@ Returns `[ElasticAPM::Error]`.
 [[api-set-tag]]
 ==== `ElasticAPM.set_tag`
 
-Add a tag to the current transaction. Tags are basic key-value pairs that are indexed in your Elasticsearch database and therefore searchable.
+Add a tag to the current transaction.
```

Comment:
Could you add a description related to the header `Context`.
Why is this not part of the section `Transaction` as it seems to only refer to Transactions and not to Errors?

(Actually is the API method `ElasticAPM.set_tag` a convenience method over ElasticAPM.current_transaction.set_tag`?)